dotcom-rendering/package.json

```diff
@@ -140,7 +141,6 @@
 		"@types/youtube": "0.0.47",
 		"@typescript-eslint/eslint-plugin": "5.61.0",
 		"@typescript-eslint/parser": "5.61.0",
-		"@yarnpkg/lockfile": "1.1.0",
```

Comment:
Can you also remove `@types/yarnpkg__lockfile`?

Reply:
oops, good spot ta